commands.txt

```diff
@@ -0,0 +1,74 @@
+## Commands
+This section lists command(s) run by WORKFLOW workflow
```

Comment:
As I mentioned elsewhere, this is a placeholder (WORKFLOW needs to be changed to ichorCNA)